lang/syntax.md

```diff
@@ -6,7 +6,7 @@ First, the general structure of programs and functions:
 ```rust
 /// Some opaque type of function names.
 /// The details of this this is represented to not matter.
-type FnName;
+use specr::FnName;
```

Comment:
Ah those will be tricky. I don't think we want specr to know which kinds of names we have. But in Coq, finding a good representation of names will be a key design choice.

I think we should either write this as
```rust
struct FnName(specr::Name);
```
or as something like
```rust
#[specr::name]
type FnName;
```
so that the tool do whatever magic it needs to do for variable names.